packages/migrate/package.json

```diff
@@ -21,22 +21,25 @@
     "src"
   ],
   "scripts": {
-    "build": "yarn tsc -b",
-    "lint": "yarn npm-run-all lint:*",
-    "lint:eslint": "yarn eslint --fix . --ext .ts",
-    "lint:tsc-src": "yarn tsc --noEmit",
+    "build": "tsc -b",
+    "lint": "npm-run-all lint:*",
+    "lint:eslint": "eslint --fix . --ext .ts",
+    "lint:tsc-src": "tsc --noEmit",
     "prepare": "yarn build",
-    "test": "yarn nyc mocha"
+    "test": "vitest --run"
+  },
+  "dependencies": {
+    "@rehearsal/reporter": "^0.0.27",
+    "@rehearsal/service": "^0.0.27",
+    "@rehearsal/shared": "^0.0.27",
+    "@rehearsal/utils": "^0.0.27",
+    "winston": "^3.6.0"
   },
   "devDependencies": {
-    "@types/chai": "^4.3.0",
     "@types/eslint": "^8.4.1",
-    "@types/mocha": "^9.1.0",
     "@types/node": "^18.0.3",
     "@types/react": "^18.0.9",
-    "chai": "^4.3.6",
-    "mocha": "^10.0.0",
-    "nyc": "^15.1.0"
+    "vitest": "^0.16.0"
```

Comment:
Why vitest? I thought to migrate all packages to something... probably vitest is the good candidate?

Reply:
Mark suggested it as for an alternative testing tool. I used it in the `migration-graph` library.

It's pretty great.
- Does easy compilation of all `.ts` files.
- Allows for all test files to be in `.ts`
- Does hot-reload of test files with the watcher in dev mode.
- The API is compatible with jest tests as well. 

In general I think this will be a great library to migrate to for internal `play/glimmer` applications since a large portion of them use Jest.

Reply:
Cool. I'll migrate other packages slowly then :)

Reply:
Eventually we should migrate all packages to vitest.